static/app/utils/metrics/mri.tsx

```diff
@@ -83,12 +83,7 @@ export function toMRI({type, useCase, name, unit}: ParsedMRI): MRI {
 }
 
 export function formatMRI(mri: MRI): string {
-  const parsedMRI = parseMRI(mri);
-  if (parsedMRI?.type !== 'v') {
-    return parsedMRI?.name;
-  }
-
-  return parsedMRI.name.split('|')[0];
+  return parseMRI(mri)?.name;
```

Comment:
The special handling for the virtual MRIs is not needed as they do not encode the project id anymore.